config/providers.cfg

```diff
@@ -108,3 +138,28 @@ providers:
     c4i: false
     type: api
     sync_type: project-api
+
+  weaver:
+    url: http://${HOSTNAME}:4001
+    title: weaver-ogc-api-processes
+    public: true
+    c4i: false
+    type: api
+    sync_type: api
+    hooks:
+      - type: request
+        path: "/processes/[w+_-]/jobs"
```

Comment:
Does the regex segment only accept a single character here? (Same thing for other similar regex below)

Reply:
Well seen. This regex is completely wrong. Fixed.